locust/configuration.py

```diff
@@ -58,11 +58,11 @@ def read_json(self):
                 self.config_data = json.load({})
         return self.config_data
 
-    def update_json_config(self, json_added, json_path, options, list_column, config_text):
+    def update_json_config(self, data_json, json_added, json_path, options, list_column, config_text):
         """
         Write JSON file configuration
         """
-        data = literal_eval(config_text)
+        data = data_json
```

Comment:
not directly for undefined jsonpath, must create json's data first

Reply:
what I mean is why don't use the data_json directly. since it this method, the data_json is not being used. rather than creating new variable with the same value